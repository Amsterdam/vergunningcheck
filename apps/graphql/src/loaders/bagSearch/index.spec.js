const rewire = require("rewire");
const bagSearch = require(".");

const rewired = rewire(".");
const { reducer } = rewired.__get__("loader");

describe("bagsearch", () => {
  test("reducer", async () => {
    expect(reducer(require("./__mocks__/basisweg30.json"))).toMatchSnapshot();
  });

  test("findAddress", async () => {
    const x = await bagSearch.load(["basisweg 30"]);
    expect(await x[0]).toMatchSnapshot();
  });
});
