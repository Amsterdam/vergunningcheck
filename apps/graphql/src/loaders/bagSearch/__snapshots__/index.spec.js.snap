// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bagsearch findAddress 1`] = `
Array [
  Object {
    "_adressableObjectId": "0363010000578090",
    "houseNumber": 30,
    "houseNumberFull": "30",
    "houseNumberLetter": null,
    "houseNumberSuffix": null,
    "postalCode": "1043AP",
    "residence": "Amsterdam",
    "streetName": "Basisweg",
    "type": "BUILDING",
  },
]
`;

exports[`bagsearch reducer 1`] = `
Object {
  "_adressableObjectId": "0363010000578090",
  "houseNumber": 30,
  "houseNumberFull": "30",
  "houseNumberLetter": null,
  "houseNumberSuffix": null,
  "postalCode": "1043AP",
  "residence": "Amsterdam",
  "streetName": "Basisweg",
  "type": "BUILDING",
}
`;
